build/configs/artik05x/artik05x_download.sh

```diff
@@ -78,17 +79,32 @@ download()
         exit 1
     fi
 
+    # check existence of romfs.img if romfs is enabled
+    if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+        pushd ${OS_DIR_PATH} > /dev/null
+        sh ../tools/fs/mkromfsimg.sh
+        if [ ! -f "${FW_DIR_PATH}/romfs.img" ]; then
+            echo "ROMFS image is not present"
+            exit 1
+        fi
+        popd > /dev/null
+    fi
+
     # Download all binaries using openocd script
+    OPENOCD_CMD="flash_protect off; \
+                 flash_write bl1 ${FW_DIR_PATH}/bl1.bin; \
+                 flash_protect on; \
+                 flash_write bl2 ${FW_DIR_PATH}/bl2.bin; \
+                 flash_write sssfw ${FW_DIR_PATH}/sssfw.bin; \
+                 flash_write wlanfw ${FW_DIR_PATH}/wlanfw.bin; \
+                 flash_write os ${TIZENRT_BIN}; "
+    if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+    OPENOCD_CMD=${OPENOCD_CMD}"set romfs_partition_enable 1; \
```

Comment:
in CONFIG_FS_ROMFS != "y" case,
Don't we need to call "set romfs_parition_enable 0"?